gradle/scripts/version.gradle

```diff
@@ -1,5 +1,5 @@
 ext.getBuildId = {
-    return (project.hasProperty('buildId') && !project.buildId.isEmpty()) ? project.buildId : 'dev'
+    return project.findProperty('buildId') ?: 'dev'
```

Comment:
Groovy considers both `null` and empty strings falsey, so the original logic is preserved.